### PR TITLE
#9 vagrant

### PR DESCRIPTION
**Pull request by YufeiGao6**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      # Create a Python3 Virtual Environment and Activate it in .profile
+      sudo -H -u vagrant sh -c 'python3 -m venv ~/venv'
+      sudo -H -u vagrant sh -c 'echo ". ~/venv/bin/activate" >> ~/.profile'
+      sudo -H -u vagrant sh -c '. ~/venv/bin/activate && cd /vagrant && pip install -r requirements.txt'
+    SHELL
+  
+    ######################################################################
+    # Add PostgreSQL docker container
+    ######################################################################
+    # docker run -d --name postgres -p 5432:5432 -v psql_data:/var/lib/postgresql/data postgres
```
Comment (rutvik2151) [inline Vagrantfile:71]: Are we using Postgres as the Database?

**Review by shivkhattar — APPROVED** [review]

File: Vagrantfile (hunk 1 above)
Comment (shivkhattar) [reply Vagrantfile:71]: Yes!

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    config.vm.provision :docker do |d|
+      d.pull_images "postgres:alpine"
+      d.run "postgres:alpine",
```
Comment (shivkhattar) [inline Vagrantfile:74]: @YufeiGao6 Did you try running the vagrant environment and checking that postgres is actually coming up?

Comment (YufeiGao6) [reply Vagrantfile:74]: I did vagrant up, everything seems fine to me. But I don't know how to check postgres.

Comment (shivkhattar) [reply Vagrantfile:74]: After doing ```vagrant ssh```, write the command ```docker images``` to see what docker images are present. To check logs of a a particular container, run ```docker container logs {containerName}```, for our case ```docker container logs postgres```. If the logs don't report any errors, we're good to go.

Comment (YufeiGao6) [reply Vagrantfile:74]: I did this:

`YufeideMacBook-Pro:orders yufeigao$ postgres --help
postgres is the PostgreSQL server.
Usage:
  postgres [OPTION]...`

postgres should be coming up.

Comment (shivkhattar) [reply Vagrantfile:74]: After doing ```vagrant ssh``` run this command to get into the database: ```docker exec -ti postgres psql -U postgres```

**Review by shivkhattar — APPROVED** [review]

**Review by xiaoshi22 — APPROVED** [review]

File: Vagrantfile (hunk 2 above)
Comment (YufeiGao6) [reply Vagrantfile:74]: Yes, it worked. I'll merge this pull request.

**Pull request merged**